### PR TITLE
update readme live

### PR DESCRIPTION
**Pull request by deadxears**
updated live reload

### CODE REVIEW

**Review by HeroesOfOnesAndZeros — CHANGES_REQUESTED** [review]
- Remove old vno run dev section
- provide "OR" deno run... way to run the run dev command like the previous run dev section, similar to other commands

File: README.md [hunk 1]
```diff
@@ -169,6 +169,16 @@ deno run --allow-read --allow-write --allow-net --unstable https://deno.land/x/v
 
 ![vno run dev](https://i.ibb.co/RckD0Tm/vno-run-dev.gif)
 
+# Live Reload
+
+- Native vno run dev command automatically enables live reload
```
Comment (unknownbreaker) [inline README.md:174]: Change vno run dev to inline code block. Use back ticks

File: README.md [hunk 2]
```diff
@@ -169,6 +169,16 @@ deno run --allow-read --allow-write --allow-net --unstable https://deno.land/x/v
...
+- Note: live reload injects a websocket connection to build.js, to remove it use vno run build
```
Comment (unknownbreaker) [inline README.md:175]: Change "websocket" to "WebSocket"

**Review by unknownbreaker — CHANGES_REQUESTED** [review]
I left a few comments

File: README.md (hunk 1 above)
Comment (deadxears) [reply README.md:174]: not sure what you mean by inline code block?

File: README.md (hunk 2 above)
Comment (deadxears) [reply README.md]: will update still a bit confused on first Rob comment

**Pull request closed without merging**